app/docker/models/container.js

```diff
@@ -91,6 +91,14 @@ export function ContainerStatsViewModel(data) {
     this.CPUCores = data.cpu_stats.cpu_usage.percpu_usage.length;
   }
   this.Networks = _.values(data.networks);
+  if (data.blkio_stats !== undefined) {
+    //TODO: take care of multiple block devices
+    this.BytesRead = data.blkio_stats.io_service_bytes_recursive[0].value;
+    this.BytesWrite = data.blkio_stats.io_service_bytes_recursive[1].value;
```

Comment:
A few times i've encountered the `data.blkio_stats.io_service_bytes_recursive` to be an empty list; which prevents all the other graph data to be rendered/displayed.
A safer approach would be to detect the presence of op codes and obtain the value if they exist:
```javascript
var readData = data.blkio_stats.io_service_bytes_recursive.find(d => d.op === "Read");
if(readData !== undefined) {
  this.BytesRead = readData.value;
}
var writeData = data.blkio_stats.io_service_bytes_recursive.find(d => d.op === "Write");
if(writeData !== undefined) {
  this.BytesWrite = writeData.value;
}
```

Reply:
@DarkAEther is this approach ok with you too?

Reply:
Sorry for the really late reply.
@zees-dev Thanks a lot for the review. I agree, it would be best to check for this before reading the value.
@sbusso Yes, this approach appears good to me!

Reply:
I also noticed that #5017 has been opened with the change included. Thank you @zees-dev!

Reply:
@DarkAEther @zees-dev @sbusso  Apparently, the values under `data.blkio_stats` can also be `null` in some cases. This seems to be norm on Windows (e.g. [1](https://neilkillen.com/2019/12/15/monitoring-sitecore-containers/), [2](https://github.com/influxdata/telegraf/issues/3789#issuecomment-366361909)), but I'm encountering it also on a current Arch Linux with ZFS.

Whatever the cause (I'm still investigating), I'm thinking if handling such cases gracefully wouldn't be a good idea as well. Maybe extend the condition on line [94](https://github.com/portainer/portainer/blob/6e9f47272350962434a7d3a579570fe1e1750681/app/docker/models/container.js#L94) to something like this?
```js
if (data.blkio_stats !== undefined && data.blkio_stats.io_service_bytes_recursive !== null) {
```